airbyte-integrations/connectors/source-monday/source_monday/spec.json

```diff
@@ -4,14 +4,81 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Monday Spec",
     "type": "object",
-    "required": ["api_token"],
-    "additionalProperties": false,
+    "required": [],
+    "additionalProperties": true,
     "properties": {
-      "api_token": {
-        "type": "string",
-        "description": "This is the API token to authenticate requests to Monday. Profile picture (bottom left) => Admin => API",
-        "airbyte_secret": true
+      "credentials": {
+        "title": "Authorization Method",
+        "type": "object",
+        "oneOf": [
+          {
+            "type": "object",
+            "title": "OAuth2.0",
+            "required": ["access_token"],
+            "properties": {
+              "auth_method": {
+                "type": "string",
+                "const": "oauth2.0",
+                "enum": ["oauth2.0"],
+                "default": "oauth2.0",
+                "order": 0
+              },
+              "client_id": {
```

Comment:
if client_Id and client secret are not used inside of connectors itself then we do not need it here